pkg/odo/genericclioptions/runnable.go

```diff
@@ -81,6 +83,13 @@ func GenericRun(o Runnable, cmd *cobra.Command, args []string) {
 
 	// LogErrorAndExit is used so that we get -o (jsonoutput) for cmds which have json output implemented
 	util.LogErrorAndExit(checkConflictingFlags(cmd, args), "")
+
+	deps, err := dependencies.Fetch(cmd)
+	if err != nil {
+		util.LogErrorAndExit(err, "")
+	}
+	o.SetDependencies(deps)
```

Comment:
How about using "clientset" instead of "dependencies"? Suggesting this because I have come across it in other code bases.